src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1711,8 +1715,6 @@ protected internal override void OnProjectRemoved(ProjectId projectId)
                 }
             }
 
-            base.OnProjectRemoved(projectId);
```

Comment:
if we're not calling this here anymore, where is it calleD?  i think i missed it.

Reply:
Called here now: https://github.com/dotnet/roslyn/pull/62257/files/bc2c117a9e2879ca5c696298e8e829d59433a3c7#diff-ed848e0f1ffd29eb5f329de7a11007a2801075fa3173f8b25dc30ad342903606R1244

Which I think answers your other question too. Before we just called the overridden method, which did some cleanup and then deferred to the base. Now we call the cleanup, but let the caller call the appropriate On* method.